README.md

```diff
@@ -61,7 +62,38 @@ For eg. With the following configuration
 }
 ```
 
-and `[README](README.html)` as input, this rule will check for the existance of `README.md` file.
+and `[README](README.html)` as input, this rule will check for the existence of `README.md` file.
+
+### route-map
+This option takes an array of source and destination pairs. The source value is a Regex, and the destination value is a 
+String. However, the destination value can include capture groups using a "\" but remember to add an extra "\" to ensure 
+the JSON formatting is valid. If a link contains the source value, then the portion of the link that matches the source
+value is replaced with the destination value.
+
+
+For e.g. With the following configuration
+```json
+{
+    "rules": {
+        "no-dead-relative-link": {
+            "route-map": [
+                {
+                  "source": "^(../)\\1javadocs/",
+                  "destination": "../website/static/javadocs/"
+                },
+                {
+                  "source": "^../javadocs/(\\d+\\.{1}\\d+\\.{1}\\w+)",
+                  "destination": "../../static/javadocs/\\1"
+                }
+            ]
+        }
+    }
+}
+
+```
+
+and `../../javadocs/overview-summary.html` as input, this rule with check for the existence of the `overview-summary.html`
+file at `../website/static/javadocs/overview-summary.html`.
```

Comment:
I think this example needs to be clarified. There are 2 separate route-maps mentioned but there's explanation for only one of them. What if the example is separated?